### ISSUE TITLE
Cannot bind to Ctrl-minus

### ISSUE THREAD

**Issue by magnouvean** [ahrm/sioyek#426]
I prefer zooming with control modifier (C - +/-), and while mapping in keys.config works great for C-+, I'm having trouble binding to C--. In the terminal I get the following log:
```
Warning: invalid command (key:-); resetting to root
```
Is it possible to map to ctrl-minus? Would be great to have this feature :)
For info, I am using a non-us keyboard layout (if that's relevant)

**Comment by jmburgos** [comment]
I am having a similar issue with the zoom_in.  I prefer zooming with the + and - keys.  In my keys.config I have   

```
zoom_in             +
zoom_out            -
```
With this, zoom_out works fine, but zoom_in does not.  When I press the "+" key I get the same warning.

I am also using a non-us keyboard.

Strangely, when I press ":" to open the list of commands, the keybinding associated to zoom_in appears as "<S-+>".

**Comment by ahrm** [comment]
The bug with control+minus should be fixed in 57e4003cc77b48a9bcca48fdcd5c8b9af4a90291.
